src/fiber.cr

```diff
@@ -1,6 +1,10 @@
 require "crystal/system/thread_linked_list"
 require "./fiber/context"
 
+{% if flag?(:linux) || flag?(:solaris) %}
```

Comment:
Hmm. On one hand I kinda wish for combinations of flags to be composable so that it is possible to see *why* the logic is differenton these platforms. on the other hand it is nice to see where it apply. :shrug:

Reply:
They'll be replaced with `:ev_epoll` before merge anyway.